pkg/api/v1/types.go

```diff
@@ -218,6 +218,8 @@ const (
 	NamespaceDefault string = "default"
 	// NamespaceAll is the default argument to specify on a context when you want to list or filter resources across all namespaces
 	NamespaceAll string = ""
+	// NamespaceSystem is the system namespace where we place system components.
+	NamespaceSystem string = "kube-system"
```

Comment:
Is this still needed now that you have SchedulerDefaultLockObjectNamespace ? 

It looks like this is unused.